modules/bitgo/test/v2/unit/wallet.ts

```diff
@@ -2323,6 +2323,31 @@ describe('V2 Wallet:', function () {
         intent.recipients![0].tokenData!.should.have.property('decimalPlaces', recipients[0].tokenData.decimalPlaces);
       });
 
+      it('should populate intent with calldata', async function () {
+        const recipients = [
+          {
+            address: '0x101c3928946b2e1d99759e8e5d34b5e94c1a8e2f',
+            amount: '0',
+            data: '0x000011112222',
+          },
+        ];
+
+        const mpcUtils = new ECDSAUtils.EcdsaUtils(bitgo, bitgo.coin('gteth'));
+        // @ts-expect-error only pass in params being tested
```

Comment:
Are there any ways that we can not suppress this error? Seems like a bit of shortcutting compilation errors here

Reply:
copied pasted from the other unit tests above

Reply:
ok, created a tracking ticket at BG-60911